docs/code-quality/ca1034.md

```diff
@@ -30,7 +30,7 @@ ms.workload:
 
 ## Cause
 
-An externally visible type contains an externally visible type declaration. Nested enumerations and protected types are exempt from this rule.
+An externally visible type contains an externally visible type declaration. Nested enumerations, protected types and builder patterns are exempt from this rule.
```

Comment:
Do you want to explain what is the builder pattern? Do you want to list the exact allowance conditions?

Reply:
I think it is probably intuitive enough. We can add it if we get feedback about it not being clear.